Sources/PointFree/Blog/BlogPosts/BlogPost0026_html020.swift

```diff
@@ -0,0 +1,99 @@
+import Foundation
+
+let post0026_html020 = BlogPost(
+  author: .pointfree,
+  blurb: """
+Announcing swift-html 0.2.0: support for CocoaPods, Carthage, SnapshotTesting, and more!
+""",
+  contentBlocks: [
+    .init(
+      content: """
+Today we're releasing our [first minor update](https://github.com/pointfreeco/swift-html/releases/tag/0.2.0) to [swift-html](\(gitHubUrl(to: .repo(.html)))).
+
+It contain a number of new features and fixes, including patches from the community!
```

Comment:
can will fill this out a bit more? remind people what the lib is, and maybe combine these sentences. otherwise the post looks very meek.